components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts

```diff
@@ -261,12 +263,27 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
     });
     observableForkJoin(installProfiles).pipe(
       takeUntil(this.isDestroyed))
-      .subscribe(
-        () => this.refreshProfiles(),
-        () => this.showDownloadError()
+      .subscribe(data => {
+        if (data) {
+          this.refreshProfiles();
+          this.showDownloadError();
```

Comment:
I'm quite certain that this isn't correct. Before, it was
```js
      .subscribe(
        () => this.refreshProfiles(),
        () => this.showDownloadError()
```
so, there's *two* callbacks given to `subscribe`, and the second one is the error callback.

Now, we're giving *two* callbacks to `subscribe`, the error callback (second one) is `(error) => this.showAuthError(error)`, and the success callback is
```js
        if (data) {
          this.refreshProfiles();
          this.showDownloadError();
       }
```
**So, we'll always be displaying a download error?**

Can you verify this code change with a successful profile download? It should *not* display that download error.

Reply:
@srenatus  Yes you were right I have updated my changes accordingly.